samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -439,6 +445,7 @@ static int time_since_presence_detected_get(struct bt_mesh_sensor_srv *srv,
 
 		err = bt_mesh_sensor_value_from_micro(format, micro_since_detect, rsp);
 		if (err == -ERANGE) {
+			printk("Time since presence detected is out of range\n");
```

Comment:
```suggestion
			printk("Warning: Time since presence detected is out of range and was clamped\n");
```

Otherwise it could look like an error occured and that the server couldn't publish the time.

Reply:
I think this is fine now, however, I should point out that this early return will cause the `Time since presence detected(%d): %s\n` to not be printed if the value is clamped, if you hadn't noticed. Approved anyways now.

Reply:
(Can be fixed easilly with replacing `return 0` with `err = 0`)

Reply:
Fixed